vizro-core/src/vizro/actions/_action_manager/_action_manager.py

```diff
@@ -0,0 +1,40 @@
+"""The action manager handles creation of all required actions callbacks."""
+
+from vizro.actions._action_manager._build_action_loop_callbacks import _build_action_loop_callbacks
+from vizro.actions._action_manager._get_action_loop_components import _get_action_loop_components
+from vizro.managers import model_manager
+from vizro.models import Action
+
+
+class ActionManager:
+    @classmethod
+    def build(cls):
+        """Builds callbacks for the action loop and for each Action in the Dashboard and returns their components.
+
+        Returns:
+            All required components for the action loop and for each Action in the Dashboard.
+        """
+        return cls._build_action_loop() + cls._build_action_models()
```

Comment:
So sorry, have to correct myself here! Given that the output are action components hidden in the dashboard container,  it might be more suitable to call this `_get_all_action_components`. What do you think? Leave as is if you don't think it's an improvement 😄

Reply:
Let's continue the conversation about naming [here](https://github.com/mckinsey/vizro/pull/11#discussion_r1317593612)